packages/website/pages/docs/how-to/get-status.md

```diff
@@ -0,0 +1,82 @@
+
+The **Pin Status** column shows the status of the pinning operation. There are three possible values for the Pin Status column: `Queuing`, `Pinned`, and `Failed`.
+
+The `Queuing` status indicates that a request to pin the data has been submitted to a cluster node, and the node has added the request to a queue for pinning. Depending on how busy the cluster is, an upload may stay in the `Queuing` state for some time before transitioning to `Pinned` or `Failed`.
+
+A `Pinned` status means that the cluster has successfully pinned the data and will retain a copy until instructed otherwise.
+
+A `Failed` status indicates that an operation failed while attempting to pin the data. Note that a `Failed` status may be temporary, as the cluster will continually retry `Failed` pins in the background and will update the status to `Pinned` if a retry attempt is successful.
+
+There are several reasons why an upload may end up in the `Failed` state.
+
+If you are able to [retrieve the data from IPFS][howto-retrieve], the cluster is likely waiting to retry a previous attempt and will soon pin the data.
+
+If you uploaded your data directly to the NFT.Storage service using a [client library][client-js], [the HTTP API][client-http], or the [Upload page on the website][upload-page], a `Failed` status may indicate that one or more chunks of data was not recieved by the service. This may happen when uploading large files or directories containing many files and may be resolved by retrying the upload.
+
+If your data was pinned using the Pinning Service API, a `Failed` status usually means that the cluster wasn't able to fetch the complete content over the IPFS peer-to-peer network. These issues can be hard to diagnose due to the many variables involved, especially when original source of the data may have an unreliable connection to the network or is behind a firewall. If the data is being provided by another pinning service such as Pinata, or if you've verified that the data is retrievable using IPFS (ideally from multiple distinct locations), please [file an issue][new-issue] or join the `#nft-storage` channel on the [IPFS Discord][support-discord] so we can dig into the cause.
+
+Please note that the status reported by the NFT.Storage service may not always be in sync with the actual status as reported by the IPFS cluster. The NFT.Storage service checks if `Failed` pins have been found and pinned by the cluster once per week for up to one month after the initial upload. 
+
+If you find that your file is retrievable but the pin status still says `Failed`, please [file an issue][new-issue] with the relevant CIDs and we'll take a look. If you'd prefer to keep the CIDs private, reach out to one of the devs on the `#nft-storage` [Discord channel][support-discord] in a direct message.
```

Comment:
Wouldn't tell them to DM right away if they don't want to share the CID - they can file an issue letting us know they don't want to share the CID and we'll let them know how to proceed.